### ISSUE TITLE
My Model

### ISSUE THREAD

**Issue by mosqueteiro** [mosqueteiro/detecting_trafficlights#23]
* [x] Run model
* [x] Update model layers for overfitting
* [x] move train_test_split outside
* [x] update model structure parameters
* [x] change learning rate
* [x] rerun model with less epochs while tuning

**Comment by mosqueteiro** [comment]
There are some issues with the data gen and doing the train test split in there. moving it outside of data gen

**Comment by mosqueteiro** [comment]
* [x] lower LR
* [x] change activation to tanh

**Comment by mosqueteiro** [comment]
tanh didn't help, rerunning with relu and he_uniform initializer
